packages/assets-controllers/src/selectors/token-selectors.ts

```diff
@@ -224,7 +240,12 @@ const selectAllEvmAssets = createAssetListSelector(
       ) as [Hex, Token[]][]) {
         for (const token of addressTokens) {
           const tokenAddress = token.address as Hex;
-          const accountGroupId = accountsMap[accountAddress];
+          const account = accountsMap[accountAddress];
```

Comment:
This should not be possible, but better not to take any chances that can crash the app.